data/json/monsters/mutant.json

```
@@ -43,7 +43,7 @@
     "anger_triggers": [ "PLAYER_WEAK", "PLAYER_CLOSE", "HOSTILE_SEEN", "SOUND" ],
     "fear_triggers": [ "HURT", "FIRE" ],
     "death_drops": "mon_mutant_experimental_death_drops",
-    "upgrades": { "half_life": 23, "into_group": "GROUP_MUTANT_EVOLVED" },
+    "upgrades": { "half_life": 72, "into_group": "GROUP_MUTANT_EVOLVED" },
```

```suggestion
    "upgrades": { "half_life": 36, "into_group": "GROUP_MUTANT_EVOLVED" },
```
I agree to a slight bump, but they are found in highly desirable/intendedly dangerous areas so I wouldn't have half the base mutants go up against a December survivor.